patches/ajv+6.12.5.patch

```diff
@@ -1,25 +0,0 @@
-diff --git a/node_modules/ajv/lib/ajv.d.ts b/ajv.d.ts
-index cc2881b3..a5ef48ce 100644
---- a/node_modules/ajv/lib/ajv.d.ts
-+++ b/node_modules/ajv/lib/ajv.d.ts
-@@ -201,6 +201,7 @@ declare namespace ajv {
-     cache?: object;
-     logger?: CustomLogger | false;
-     nullable?: boolean;
-+    xNullable?: boolean;
```

Comment:
If we wanted to make this patch work, i think the main difference is that nullable is always set to true now, which is funny because its been removed in OAS3.1, but... whatever that's the change according to https://github.com/ajv-validator/ajv/releases/tag/v7.0.0